src/main/fc/rc.c

```diff
@@ -496,134 +496,182 @@ NOINLINE void initAveraging(uint16_t feedforwardAveraging)
     }
 }
 
+#ifdef USE_FEEDFORWARD
 FAST_CODE_NOINLINE void calculateFeedforward(const pidRuntime_t *pid, int axis)
 {
     const float rxInterval = currentRxIntervalUs * 1e-6f; // seconds
-    float rxRate = currentRxRateHz;
-    static float prevRxInterval;
+    float rxRate = currentRxRateHz;                       // 1e6f / currentRxIntervalUs;
 
     static float prevRcCommand[3];
     static float prevRcCommandDeltaAbs[3];          // for duplicate interpolation
     static float prevSetpoint[3];                   // equals raw unless interpolated 
     static float prevSetpointSpeed[3];
     static float prevAcceleration[3];               // for duplicate interpolation
     static bool prevDuplicatePacket[3];             // to identify multiple identical packets
-    static uint16_t feedforwardAveraging = 0;
-
+    static uint16_t feedforwardAveraging;
     if (feedforwardAveraging != pid->feedforwardAveraging) {
         feedforwardAveraging = pid->feedforwardAveraging;
         initAveraging(feedforwardAveraging);
     }
 
-    const float rcCommandDeltaAbs = fabsf(rcCommand[axis] - prevRcCommand[axis]);
+    const float rcCommandDelta = rcCommand[axis] - prevRcCommand[axis];
     prevRcCommand[axis] = rcCommand[axis];
+    float rcCommandDeltaAbs = fabsf(rcCommandDelta);
 
-    float setpoint = rawSetpoint[axis];
-    float setpointSpeed = (setpoint - prevSetpoint[axis]);
+    const float setpoint = rawSetpoint[axis];
+    const float setpointDelta = (setpoint - prevSetpoint[axis]);
```

Comment:
No need for parentheses
```suggestion
    const float setpointDelta = setpoint - prevSetpoint[axis];
```

Reply:
Yes, thanks.